HW3_Bar/src/main/java/com/hillel/hw3/Bar.java

```diff
@@ -0,0 +1,39 @@
+package com.hillel.hw3;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+public class Bar {
+
+    private final List<String> orders = new ArrayList<>();
+
+    private final List<String> drinks = Arrays.asList("Beer", "Whisky", "Coffee", "Juice");
+
+
+    public synchronized String getDrink() {
+        return drinks.get(new Random().nextInt(drinks.size()));
```

Comment:
You can use a better approach with concurrent collection instead of synchronized method